platform/pulpcore/tasking/tasks.py

```diff
@@ -64,7 +64,7 @@ def _queue_reserved_task(name, inner_task_id, resource_id, inner_args, inner_kwa
             concurrently with yours.
         inner_args (tuple): The positional arguments to pass on to the task.
         inner_kwargs (dict): The keyword arguments to pass on to the task.
-        options (dict): For all options accepted by apply_async please visit: http://docs.celeryproject.org/en/latest/reference/celery.app.task.html#celery.app.task.Task.apply_async  #NOQA
+        options (dict): For all options accepted by apply_async please visit: http://docs.celeryproject.org/en/latest/reference/celery.app.task.html#celery.app.task.Task.apply_async  # NOQA
```

Comment:
since we use lowercase `#noqa` elsewhere, it would be better to keep it consistent.